component/service/src/main/java/org/exoplatform/social/rest/impl/user/UserRestResourcesV1.java

```diff
@@ -847,6 +847,57 @@ public Response sendOnBoardingEmail(@Context HttpServletRequest request,
     sendOnBoardingEmail((UserImpl) user, url);
     return Response.ok().build();
   }
+
+  @POST
+  @Path("multiSelection/{action}")
+  @Produces(MediaType.APPLICATION_JSON)
+  @RolesAllowed("administrators")
+  @ApiOperation(value = "Onboard, enable ou disable a specific users", httpMethod = "POST", response = Response.class, notes = "This will enroll, enable ou disable a specific users.")
+  public Response multiSelectionAction(@Context HttpServletRequest request,
+                                       @ApiParam(value = "Action", required = true) @PathParam("action") String action,
+                                       @ApiParam(value = "User List", required = true) List<String> users) throws Exception {
+
+    switch (action) {
+    case "onboard":
+      StringBuilder url = new StringBuilder();
+      url.append(request.getScheme()).append("://").append(request.getServerName());
+      if (request.getServerPort() != 80 && request.getServerPort() != 443) {
+        url.append(':').append(request.getServerPort());
+      }
+      PortalContainer container = PortalContainer.getCurrentInstance(request.getServletContext());
+      url.append(container.getPortalContext().getContextPath());
+      for (String username : users) {
+        UserHandler userHandler = organizationService.getUserHandler();
+        User user = userHandler.findUserByName(username);
+        if (user == null) {
+          throw new WebApplicationException(Response.Status.BAD_REQUEST);
+        }
+        sendOnBoardingEmail((UserImpl) user, url);
```

Comment:
Before sending the mail, you should check that the user is enabled and not already loggued. Perhaps this verification should be done in sendOnBoardingEmail

Reply:
Done !